### PR TITLE
Ana/add netlify file with csp headers

### LINKED ISSUE #142: CSP
add netlify CSP file

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #142

### CODE REVIEW

**Review by jbibla — CHANGES_REQUESTED** [review]
not working as expected on deploy branch

**Comment by Bitcoinera** [comment]
> not working as expected on deploy branch

OK, the comments were wrongly placed. I saw it by installing an extension in VSCode

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Add in the 'connect-src' line the RPC URLS you might need
+# For example:  https://lcd.nylira.net (Nylira's public Cosmos node)
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Content-Security-Policy = '''
+      default-src 'self' https://*.netlify.app;
+      connect-src 'self' https://lcd.nylira.net;
+      img-src 'self' data: https://*.netlify.app;
+      script-src 'self' 'unsafe-eval' https://*.netlify.app 'nonce-4AEemGbGTJptoIGFP3Nd' 'nonce-HYjB4SnB0b0lHRlAzTmQ' 'nonce-YW5hbHl0aWNzLWx1bmll';
```
Comment (faboweb) [inline netlify.toml:11]: Which scripts are the nonces referring to?

Comment (Bitcoinera) [reply netlify.toml]: yeah, I removed. It doesn't makes sense to have it here :+1:

**Comment by Bitcoinera** [comment]
You need to be patient with these CSP PRs. Lots of try & error & wait for deployment

**Comment by faboweb** [comment]
can't connect to keybase or keplr

**Pull request merged**